tutorials/18_diagnostics_lc2st.ipynb

```diff
@@ -0,0 +1,821 @@
kpqdr6iouLkZeXpzYRaSM8PByjR4+W3g8aNAgzZ840WHvItPEIvxbMzMwwcOBA6bU2ZGZm8PHrrVMdRETGSNeM1EfGElHj5u/vj7i4OLRv3x4ZGRlYtGgR+vfvj9OnTyMzMxMKhQIODg5q67i6uiIzM7PaOqOjo7Fo0SKd2qVLvskaQTauSbhQr9ub9Vg7jcqHh4fj448/RnR0NF577TVp/u7duzFmzBgIIfTdRCKdccCvBSsrKyQmJupUh0JpicjVn0j1ERGZCl0zUh8ZS0SNW0hIiPTaz88P/v7+8PLywueff671311RUVGYPXu29D4vLw+enp4a1aFLvimUlsxGI2BpaYm33noL06ZNQ9OmTQ3dHKKHMs3dg0RERERE/5+DgwPatWuHixcvws3NDSUlJcjJyVErk5WVVeU1/xWUSiXs7OzUJmp8AgMD4ebmhujo6CqXL1y4EF27dlWbt3btWrRq1aruG0dUBQ74iYiIiMik5efnIy0tDe7u7ujRowcsLCxw8OBBafn58+dx5coVBAQEGLCV1BDI5XIsX74c69evx7Vr1wzdHKKH4oBfCwUFBXB2doazszMKCgqqLHP/NUhVXY9UfLcQbz7ZB28+2Qcrvj5ZV00lIqp3D2ZkVRm4JuFCtddq1iZjiYhqMmfOHBw+fBiXLl3CL7/8gjFjxkAul+Opp56Cvb09Jk+ejNmzZ+PQoUNISUnBc889h4CAgDq/Q78u+VZ8t5DZaCTGjBmDrl27YsGCBYZuCtFD8Rp+Ld28eVPnOgpy7+ihJURExkfXjNRHxhJR43Xt2jU89dRTuHXrFpydndGvXz8kJyfD2dkZALBmzRqYmZkhNDQUxcXFCAoKwoYNG+qlbbrkG7PReLz11lsYMmQI5syZY+imENWIA34iIiIiMinbtm2rcbmlpSViYmIQExNTTy0iUzNgwAAEBQUhKioK4eHh0nwzM7NKd+svLS2t59YR/YsDfiIiIiIiIg2tWLECXbt2Rfv27aV5zs7OyMzMhBACMpkMAHDy5EkDtZDIyK/hX7hwIWQymdrk6+srLS8qKkJkZCSaNWsGW1tbhIaGIisry4AtJiIiIiKixqBz584ICwvDunXrpHmDBg3CjRs3sHLlSqSlpSEmJgbfffedAVtJjZ3RH+Hv2LEjDhw4IL03N/+3ybNmzcI333yDHTt2wN7eHjNmzMDYsWPx888/G6KpRERERESkpVmPtTN0EzS2ePFibN++XXrfoUMHbNiwAcuXL8eSJUsQGhqKOXPm4IMPPjBgK6kxM/oBv7m5eZXPRM3NzcWHH36IrVu3YsiQIQCA2NhYdOjQAcnJyXV+l1UiIiIiImo84uLiKs1r1aoViouL1ea98MILeOGFF9Tmvf7669XWk5iYqK8mElVi9AP+1NRUeHh4wNLSEgEBAYiOjkbLli2RkpKC0tJSBAYGSmV9fX3RsmVLJCUl1TjgLy4uVvti5uXladQmMzMz9OzZU3qtDZmZGTzbdZJeExGZCl0zUh8ZS0RkjHTJNxmzkYi0YNQDfn9/f8TFxaF9+/bIyMjAokWL0L9/f5w+fRqZmZlQKBRwcHBQW8fV1RWZmZk11hsdHY1FixZp3S4rKyscO3ZM6/UBQKG0xKz3vtSpDiJq3BYuXFgpy9q3b48///wTwL37nPznP//Btm3b1B475erqWqft0jUj9ZGxRETGSJd8UygtmY1EpDGjHvCHhIRIr/38/ODv7w8vLy98/vnnsLKy0rreqKgozJ49W3qfl5cHT09PndpKRGQIvM8JEREREVXHqAf8D3JwcEC7du1w8eJFPPbYYygpKUFOTo7aUf6srKwqr/m/n1KphFKprOPWEhHVPd7nhIiIiIiq06AuAMrPz0daWhrc3d3Ro0cPWFhY4ODBg9Ly8+fP48qVKwgICKjTdhQWFqJVq1Zo1aoVCgsLtaqjpOguljwzBEueGYKSort6biERNRYV9zlp3bo1wsLCcOXKFQB46H1OqlNcXIy8vDy1SVO6ZqQ+MpaIyBjpkm8lRXeZjUSkMaM+wj9nzhyMHDkSXl5euH79OhYsWAC5XI6nnnoK9vb2mDx5MmbPng1HR0fY2dnhpZdeQkBAQJ0fuRJC4PLly9Jrbeu4k/W3TnUQUeNWF/c50fUeJ4DuGamPjCUiMka65BuzkYi0YdQD/mvXruGpp57CrVu34OzsjH79+iE5ORnOzs4AgDVr1sDMzAyhoaFqN6QiImoM6uI+J7zHCREREZHpMOoB/7Zt22pcbmlpiZiYGMTExNRTi4iIjJc+7nPCe5wQERERmY4GdQ0/ERFVz1juc0JERERExoEDfiKiBmrOnDk4fPgwLl26hF9++QVjxoyp8j4nhw4dQkpKCp577rl6uc8JERFRY9eqVSusXbvW0M3QG1PrT2PCAT8RUQNVcZ+T9u3bY/z48WjWrFml+5w8/vjjCA0NxYABA+Dm5oadO3cauNVEREQN29WrV/H888/Dw8MDCoUCXl5eeOWVV3Dr1i1DN42oEqO+ht9YyWQyPPLII9Jrbetw9WqjUx1E1LgZ631OdM1IfWQsEZEx0iXfmI3G4a+//kJAQADatWuHzz77DN7e3jhz5gzmzp2L7777DsnJyXB0dKz3dpWXl0Mmk8HMjMdzSR1/I7RgbW2NM2fO4MyZM7C2ttaqDoWlFeZt/gbzNn8DhaV2d9MmIjJGumakPjKWiMgYPZhvaxIuSMuqe11BYWmFM2fOYMraXaaXjUIABQWGmTR8xGFkZCQUCgW+//57DBw4EC1btkRISAgOHDiAv//+G//973+lsv/88w+eeuop2NjYoHnz5mo74IUQWLhwIVq2bAmlUgkPDw+8/PLL0vLi4mLMmTMHzZs3h42NDfz9/ZGYmCgtj4uLg4ODA77++ms88sgjUCqV+N///gdLS0vk5OSotfmVV17BkCFDpPc//fQT+vfvDysrK3h6euLll19GQUGBtDw7OxsjR46ElZUVvL29ER8fr9
```

Comment:
I suggest to change this line into:
`axes[-1].legend(bbox_to_anchor=(1.1, .5), loc='center left')`